tox.ini

```diff
@@ -1,2 +1,3 @@
 [pycodestyle]
-select=E112,E113,E115,E116,E133,E201,E202,E203,E211,E221,E222,E225,E226,E228,E231,E241,E251,E271,E272,E275,E301,E302,E305,E714,W292,W293,W391
+select=E112,E113,E115,E116,E133,E201,E202,E203,E211,E221,E222,E226,E231,E241,E251,E271,E272,E275,E301,E302,E305,E714,W292,W293,W391
+ignore=E112,E113,E115,E116,E133,E201,E202,E203,E211,E221,E222,E225,E226,E228,E231,E241,E251,E271,E272,E275,E301,E302,E305,E714,W292,W293,W391
```

Comment:
No changes should be made to this file. In general, do not modify the existing lint rules.

Reply:
It throws these errors:- https://paste.fedoraproject.org/paste/3SbLJRjMPo2IPVfz6nqLiA.

Reply:
Yes, I know. This is because [this line](https://github.com/oppia/oppia/pull/5679#discussion-diff-219234418R1201) has been modified which shouldn't be. This isn't a docstring. See https://github.com/oppia/oppia/pull/5679#discussion_r219234418. The check you're implementing is that `There should not be a space after """ in a docstring`.

Reply:
Thanks, I will remove these changes done to the this file.